Sources/Extension/AudioNode+Extension.swift

```diff
@@ -0,0 +1,154 @@
+import AudioUnit
+
+extension AudioNode: CustomStringConvertible {
```

Comment:
`AudioNode` is a new class so I implemented a custom description. I can rename it to something else. Is it more about reimplementing a custom description for `AudioStreamBasicDescription`? It's very handy to print `mFormatFlags` as a readable flags. For example,
> mFormatFlags: 12 AudioFormatFlags(audio_IsPacked | audio_IsSignedInteger | ll_32BitSourceData | pcm_IsPacked | pcm_IsSignedInteger)

instead of just `mFormatFlags: 12`.

Reply:
Regarding custom classes in HK, that's fine. However, please keep the Extension folder for Cocoa frame-only extensions. Please move it to the bottom of the definition file.

As for AudioStreamBasicDescription, I haven't adopted it because the default behavior changes. I acknowledge that it's convenient for debugging, but we often avoid it in application-level development due to the numerous problems it can cause.

Reply:
Sounds good, I'll move it tomorrow. Can I keep `AudioStreamBasicDescription` extension while removing `description` override and rename it to something like `verboseDescription` for debug purpose?